docs/dapps/getting-started.md

```diff
@@ -8,91 +8,531 @@ import UntranslatedPageText from "@site/src/components/UntranslatedPageText";
+
+  //Make sure it match your address group
+  const accountGroup = 1
+
+  const deployed = deployments.getDeployedContractResult(accountGroup, 'TokenFaucet')
+
+  if(deployed !== undefined) {
+    const tokenId = deployed.contractId
+    const tokenAddress = deployed.contractAddress
+
+    // Submit a transaction to use the transaction script
+    // It uses our `wallet` to sing the transaction.
+    await Withdraw.execute(wallet, {
+      initialFields: { token: tokenId, amount: 1n }
+    })
+
+    // Fetch the latest state of the token contract, `mut balance` should have change
+    const faucet = TokenFaucet.at(tokenAddress)
+    const state = await faucet.fetchState()
+    console.log(state.fields)
+
+    // Fetch wallet balance see if token is there
+    const balance = await wallet.nodeProvider.addresses.getAddressesAddressBalance(wallet.account.address)
+    console.log(balance)
+  } else {
+    console.log('`deployed` is undefined')
+  }
+}
+
+// Let's perform one withdraw
+withdraw()
+```
+
+For the attentive people, you'll see something new coming from our `artifacts`: [Withdraw](https://github.com/alephium/nextjs-template/blob/main/contracts/withdraw.ral) which is a [TxScript](https://wiki.alephium.org/ralph/getting-started#txscript) required to interact with the `TokenFaucet` contract. Its code is quite simple, you can add this to you `contracts/withdraw.ral`
+
+```rust
+TxScript Withdraw(token: TokenFaucet, amount: U256) {
+    token.withdraw(amount)
+}
+```
+
+We now need to recompile our contracts to get the artifact for `Withdraw`
+
+```sh
+npx @alephium/cli@latest compile
+```
+
+The code of `token.ts` should be self-explanatory, you can read more on our [web3-sdk here](https://wiki.alephium.org/dapps/alephium-web3)
+
+You can now try to compile.
+
+```sh
+npx tsc --build .
+```
+
+OOPS, you should get an error coming from the `alephium.config.ts`, until now the config was used as a simple JSON, but now `TypeScript` want it to respect its [interface](https://github.com/alephium/alephium-web3/blob/d2b5b63cae015e843aa77b4cf484bc62a070f1d5/packages/cli/src/types.ts#L48-L62). Especially the `networks` is a record that need to contain the 3 `NetworkType`. You can try to fix it by yourself or update your `alephium.config.ts` file with:
+
+```typescript
+import { Configuration } from '@alephium/cli'
+
+export type Settings = {}
+
+const configuration: Configuration<Settings> = {
+  defaultNetwork: 'devnet',
+  networks: {
+    devnet: {
+      nodeUrl: 'http://localhost:22973',
+      networkId: 2, //Use the same as in your `~/.alephium/user.conf
+      privateKeys: ['672c8292041176c9056bb0dd1d91d34711ceed2493b5afc83f2012b27df2c559'],
+      settings: {}
+    },
+    testnet: {
+      nodeUrl: '',
+      privateKeys: [],
+      settings: {}
+    },
+    mainnet: {
+      nodeUrl: '',
+      privateKeys: [],
+      settings: {}
+    }
+  }
+}
+
+export default configuration
+```
+
+Now recompile
+
+```
+npx tsc --build .
```

Comment:
I still get the error because of an import in `0_deploy_faucet.ts`:

```
Module '"../alephium.config"' has no exported member 'Settings'. Did you mean to use 'import Settings from "../alephium.config"' instead?

  import { Settings } from "../alephium.config";
```

Reply:
It's solved if I add this in the `alephium.config.ts` file:

```ts
export type Settings = void;
```